templates/user/edit-profile.html

```diff
@@ -267,6 +267,12 @@
 {% block body %}
     <div class="centered-form">
         <form id="edit-form" action="" method="post" class="form-area">
+            {% if request.profile.is_peg %}
```

Comment:
Is this discoverable enough? Maybe this should be on the user page for the current user, so that they will see when they log in. Perhaps this should even be a notification like an announcement, or displayed in the navigation bar, or something.

Reply:
I've added the same notification to the rest of the profile pages.